docs/tutorials/1_the_ibm_quantum_account.ipynb

```diff
@@ -1190,30 +1191,34 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Job Manager <a name='job manager'></a>\n",
+    "### Long list of circuits\n",
+    "\n",
+    "You can submit a list of circuits in a single job, but IBM Quantum backends limit how many. You can query this limit using `backend.configuration().max_experiments`. \n",
+    "\n",
+    "`qiskit-ibmq-provider`, which is the predecessor of `qiskit-ibm`, has a convenience function called Job Manager that splits circuits into multiple jobs, if the number exceeds the backend limit. In `qiskit-ibm`, however, this function is now done implicitly by `backend.run()`. \n",
     "\n",
+    "If you submit a list of circuits using `backend.run()`, and the length of the list exceeds the backend limit, you will get back an [`IBMCompositeJob`](https://qiskit.org/documentation/stubs/qiskit_ibm.job.IBMCompositeJob.html) instance. This `IBMCompositeJob` implicitly splits the circuits into multiple sub-jobs and manage them for you. When the sub-jobs are finished, it collects and presents the results in a unified view. The sub-jobs are submitted sequentially, and if `IBMCompositeJob` hits the job limit error* when submitting a sub-job, it will wait for a previous job to finish before trying to submit again.\n",
     "\n",
-    "The Job Manager is another convenience function provided by `qiskit-ibm`. It splits experiments into multiple jobs based on backend restrictions. When the jobs are finished, it collects and presents the results in a unified view.\n",
+    "`IBMCompositeJob` is a subclass of `IBMJob`, and you can use it the same way you do with a traditional job. For example, you can continue to use methods like `status()` and `result()` to get the job status and result, respectively. You can also retrieve a previously executed `IBMCompositeJob` using the `backend.retrieve_job()` and `backend.jobs()` methods, like you would with traditional jobs.\n",
```

Comment:
`backend.retrieve_job()` and `backend.jobs()` have been removed, `IBMBackendService.job()` and `IBMBackendService.jobs()` are the replacements.